code/modules/surgery/implant.dm

```diff
@@ -130,6 +130,7 @@
 				affected.owner.custom_pain("You feel something rip in your [affected.name]!", 1)
 			user.drop_item()
 			affected.hidden = tool
+			target.internal_organs += tool
```

Comment:
This line makes it possible to put non-organ items, like an oxygen tank or a crowbar, into the internal_organs list, you probably want to make a check here.